WeatherApp/WeatherApp/ViewController/ViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  ViewController.swift
+//  WeatherApp
+//
+//  Created by Thomas Feng on 9/24/19.
+//  Copyright © 2019 Thomas Feng. All rights reserved.
+//
+
+import UIKit
+import Log
+
+class ViewController: UIViewController, UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        loadWeatherData()
+    }
+    
+    @IBOutlet var collectionView: UICollectionView!
+    
+    let reuseIdentifier = "forecastCell"
+    var forecasts = [Forecast]()
+    let log = Logger()
+    
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return forecasts.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
```

Comment:
Nice, in general we should look to move delegate functions to extensions just to keep MVC modeled views readable, but this is just a demo app. We'll follow that pattern in the future,